Util/include/Poco/Util/Timer.h

```diff
@@ -164,6 +164,7 @@ class Util_API Timer: protected Poco::Runnable
 	
 	Poco::TimedNotificationQueue _queue;
 	Poco::Thread _thread;
+	bool _stop;
```

Comment:
Should probably be std::atomic<bool>

Reply:
it would be nice but std::atomic is c++11 and can not be used therefore

Reply:
sure it can - 1.x is open for c++11 conditionally, develop ([2.0](https://github.com/pocoproject/poco/milestone/13)) unconditionally

Reply:
what are the conditions. just to say we can not in our project since our customer required us to stick with c++03 🤦‍♂️

Reply:
then you should stick with [1.x](https://github.com/pocoproject/poco/blob/poco-1.8.1/Foundation/include/Poco/Config.h#L27) releases

Reply:
sure that's what we do. and the PR is for the next 1.x release. isn't the develop still aiming for the next 1.x version

Reply:
no, develop will become 2.0. If you want it in [1.8.1](https://github.com/pocoproject/poco/tree/poco-1.8.1), then another pull has to be sent there, but see with @obiltschnig about the exact 1.8.1 release date